GAE/src/locale/es.properties

```diff
@@ -124,8 +122,10 @@ Could\ not\ validate\ variable\ name\ with\ the\ server = No se pudo validar el
 Create\ folders = Crear carpetas
 Create\ forms = Crear formularios
 Create\ new\ assignment = Crear una nueva asignaci�n
-Create\ new\ survey = Crear una nueva encuesta
+Create\ survey = Crear encuesta
+Create\ survey\ from\ template = Create survey from template
```

Comment:
this one is still in English

Reply:
These havent been translated yet. But they are from another issue so there is nothing changing.

Reply:
I would like to finish with this one and then handle separately.